Add require and bodyLimit options

davidebianchi opened the pull request.

pscanf commented:
#### Regarding the `bodyLimit` option

1. I would prefer not adding the option, but simply increasing the limit to something like `1gb` so the user doesn't even have to think about it (as it's done for CORS headers)

#### Regarding the `require` option

1. I think its documentation is a bit misleading: I would think I need to specify module-names in a JSON array, doing something like `--require ["module1", "module2"]`, which doesn't work. I would phrase it like **"require a module before startup, can be used multiple times"**

1. I wouldn't alias it to `requiredModules`. At most to `r` (like mocha does), but in that case I would also add - for consistency - a one-letter alias to the other options (eg `watch` -> `w`). Although that would also be inconsistent and generate confusion (`r` is short for `root` or for `require`?). I'd prefer to simply stay away from one-letter aliases

1. I'm wondering, does it work together with the `watch` option? (On file-change we need to purge `require`'s cache before re-requiring modules, maybe that breaks it in some way)

1. as it's implemented now it doesn't work for relative modules (eg `--require ./something.js`), since it resolves the module relative to the file where `require` is called, ie `src/index.js`. It'd be ideal to support that use case ([like mocha does](https://github.com/mochajs/mocha/blob/master/bin/_mocha#L175-L181)), but the implementation is not super-trivial, so maybe we can ignore it (anyway the main use case is requiring `require` hooks like `babel-register` or `ts-node/register`)

davidebianchi commented:
Thanks for the review!

**Regarding the `bodyLimit` option**

I removed bodyLimit options and set to `1gb`

**Regarding the `require` option**

1. modified README as suggested
2. removed alias from require options
3. the watch options is working and I've not checked issue
4. The main use case is require a node_modules, but we can add an issue as future enhancement
